src/store/actions/causes.js

```diff
@@ -1,9 +1,42 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+
+const dataSanitization = data => {
+  if (data) {
+    const {
+      id,
+      causes,
+      causes_heading: { heading_primary, heading_secondary }
+    } = data;
+    const sanitizedCauses = causes.map(
+      ({ title, description, raised, goal, image: { url: imgURL }, id }) => {
+        return {
+          title,
+          description,
+          raised,
+          goal,
+          imgURL,
+          id
+        };
+      }
+    );
+    return {
+      id,
+      causes: sanitizedCauses,
+      causes_heading: {
+        heading_primary,
+        heading_secondary
+      }
+    };
+  }
+  else {
+    return data
```

Comment:
if there's no data, you're returning the data, this is weird, I think we should think about returning something that we can actually control the value of, like `null` for example

Reply:
he was returning null and giving the nature of this function (only sanitize if there is a data) I asked him to return the exact same falsely datatype as is.

Reply:
this doesn't necessarily means that I am correct. i am telling why they made it like this.

Reply:
> if there's no data, you're returning the data, this is weird, I think we should think about returning something that we can actually control the value of, like `null` for example

The initial value of data is `null`  so if `data` is falsy the function will return data which is `null`